### PR TITLE
Use HTTPS ingest api url

### PR DESCRIPTION
**Pull request by samanehsan**
### Purpose
https://app.zenhub.com/workspaces/dcp-backlogs-5ac7bcf9465cb172b77760d9/issues/humancellatlas/secondary-analysis/461

---
### Changes

Use more secure https endpoint of ingest api url

---
### Review Instructions
Please make sure I didn't miss any urls!

---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [ ] This PR updated docstrings or documentation.
- [ ] This PR applied Python style guidelines, specifically followed [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) for this repo.
- [ ] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
- No follow-up discussions.

### CODE REVIEW

**Comment by samanehsan** [comment]
retest this please

**Comment by samanehsan** [comment]
retest this please

**Review by rexwangcc — APPROVED** [review]
LGTM! Note: Ingest hasn't deployed their HTTPS update to `prod` yet.

File: deploy/config_files/config.sh.ctmpl [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
     export DSS_URL="https://dss.integration.data.humancellatlas.org/v1"
     export SCHEMA_URL="http://schema.integration.data.humancellatlas.org/"
-    export INGEST_URL="http://api.ingest.integration.data.humancellatlas.org/"
+    export INGEST_URL="https://api.ingest.integration.data.humancellatlas.org/"
```
Comment (pshapiro4broad) [inline deploy/config_files/config.sh.ctmpl:47]: There are similar lines below at 70 and 93; do they need updating as well?

**Comment by rexwangcc** [comment]
retest this please

File: deploy/config_files/config.sh.ctmpl (hunk 1 above)
Comment (samanehsan) [reply deploy/config_files/config.sh.ctmpl:47]: Yes! Good catch!

Comment (samanehsan) [reply deploy/config_files/config.sh.ctmpl:47]: Done!

**Comment by rexwangcc** [comment]
The Jenkins failure has nothing to do with this PR, moving forward to merge this PR to unblock HCA DCP Ingest Service's deployment.

**Pull request merged**